javascript/javascript1/week3/noteApp.js

```diff
@@ -0,0 +1,57 @@
+// NOTE TAKING APP
+
+// Save a note
+const notes = [];
+
+function saveNote(content, id) {
+    notes.push({content, id});
+}
+
+saveNote('Pick up groceries', 1);
+saveNote('Do laundry', 2);
+console.log(notes);
+
+
+// Get a note
+function getNote(id) {
+    if  (typeof id === '' || typeof id === 'null' || typeof id !== 'number') {
+        alert(`Error: There is no note corresponding to this number. Please enter a valid number.`);
+    } else {
+        for (i = 0; i < notes.length; i++) {
+            if (notes[i].id === id) {
```

Comment:
Check find https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
I see what the find() method does, but I was not able to apply it to my exercise.